server/src/helpers/hash.password.ts

```diff
@@ -0,0 +1,7 @@
+import bcrypt from 'bcrypt';
+
+export const hashPassword = (password: string): string => {
+    const rounds = +`${process.env.ROUNDS}`;
```

Comment:
move to constant, write after import 
const ROUNDS = 12

if you need nubber, you can use Number(process.env.ROUNDS)

and pls give more understandable name